docs/components/window/frame.ejs

```diff
@@ -70,4 +70,24 @@
         </div>
       </div>
     `) %>
+
+    <p>
+      In case you need the fixed background attachment, add the <code>fixed-background</code>, and for scrolling attachment <code>scroll-background</code> class to the window (otherwise it uses <code>background-attachment: local;</code>).
```

Comment:
In my opinion, the glass non/moving effect being explained here is not clear. Something like "The glass effect is still by default, to make it look moving on scroll, add the XXX class" would be more user-friendly and easier to understand.

And then, let's rename the `fixed-background` class to `has-moving-glass`, and remove `scroll-background` since it doesn't make any difference from the `background-attachment: local`. This way, we abstract away the somewhat confusing technical detail of `background-attachment`, and keep things simple.

Reply:
I was unsure if the default background attachment was deliberate or not, so I added utility classes in cases where different attachments are desired.

The `scroll` attachment still differs from `local` in cases where the viewport changes size (for example when resizing the browser window or opening devtools), so Instead of renaming one class and removing the other, I'd rather remove both of them and just use inline styles in the example for `background-attachment` (as it's done in other places in the example code as well).